platform/darwin/src/MGLMapCamera.h

```diff
@@ -63,10 +85,46 @@ MGL_EXPORT
     The value `180` means the top of the map points due south, and so on.
  */
 + (instancetype)cameraLookingAtCenterCoordinate:(CLLocationCoordinate2D)centerCoordinate
-                                   fromDistance:(CLLocationDistance)distance
+                                 acrossDistance:(CLLocationDistance)distance
```

Comment:
I’m not sure any of those alternatives distinguishes between the hypotenuse and adjacent any more than `acrossDistance` would, but I just don’t think that distinction is very important. Consider that the original method – which I want to return to eventually, once we deprecate the incorrect implementation – calls it `fromDistance`, for simplicity and consistency with MapKit. The point is to distinguish from the opposite (altitude), which all these options accomplish to varying degrees. (The property, meanwhile, is still called `viewingDistance` – do you think it needs to be revisited?)

If `acrossDistance` doesn’t work, then my next suggestion would be `-cameraLookingAtDistance:toCoordinate:pitch:heading:` (`camera(lookingAt:to:pitch:heading:)`) or something like that. But honestly, I keep getting frustrated because all the good ways to express this concept in English require putting a word _after_ the quantity (e.g., “from … away”), which doesn’t work here.

Reply:
Yep - I totally hear you about language frustrations. I think the truth is that we *have* the best one, it's just that we need to deprecate it first before re-using it! 🔀 

Let's get this one in 👍

Reply:
Thanks, for what it’s worth, there is a more detailed explanation of this parameter a few lines further up:

https://github.com/mapbox/mapbox-gl-native/blob/44aa4f4be2bd504da982ffdcc0ffa5e84b2c6f31/platform/darwin/src/MGLMapCamera.h#L71-L73